src/scripts/main.js

```diff
@@ -1 +1,15 @@
 'use strict';
+
+const thumbsList = document.getElementById('thumbs');
+const largeImg = document.getElementById('largeImg');
+
+thumbsList.addEventListener('click', (e) => {
+  if (e.target.tagName === 'IMG' || e.target.tagName === 'A') {
+    e.preventDefault();
+
+    const clickedImgSrc =
+      e.target.tagName === 'IMG' ? e.target.parentNode.href : e.target.href;
```

Comment:
The current logic checks if the clicked element is an 'IMG' or 'A' tag. While this works, it might be more robust to specifically target the 'A' tags since they are the direct parents of the 'IMG' tags. This way, you can simplify the logic by always using `e.target.href` and avoid checking the tag name.